homeassistant/components/template/cover.py

```diff
@@ -270,8 +270,11 @@ def _update_tilt(self, result):
             self._tilt_value = state
 
     @property
-    def is_closed(self) -> bool:
+    def is_closed(self) -> bool | None:
         """Return if the cover is closed."""
+        if not self._position_template and not self._template and not self._optimistic:
+            return None
+
```

Comment:
Should this check instead be:
```suggestion
        if self._position is None:
            return None
```